.circleci/config.yml

```
@@ -376,6 +385,41 @@ commands:
             ICONS=pro yarn build-storybook
             ls -lRa ./storybook-build
 
+  e2e_build:
+    parameters:
+      test_runner:
+        type: enum
+        enum:
+          - testcafe
+          - playwright
+    steps:
+      - restore_repo
+      - setup_remote_docker:
+          version: *remote_docker_version
+          docker_layer_caching: false
+      - run:
+          name: Build and push E2E << parameters.test_runner >> image
+          working_directory: *workspace_compose
+          command: |
+            ecr-login
+            # The image is not expected to chage too often so we use :latest for all cache
+            E2E_IMAGE="307238562370.dkr.ecr.eu-west-1.amazonaws.com/evaka/e2e-<< parameters.test_runner >>:latest"
+            docker pull "$E2E_IMAGE"
+            docker build -t "$E2E_IMAGE" --cache-from "$E2E_IMAGE" -f "./e2e/<< parameters.test_runner >>.Dockerfile" ./e2e/
+            docker push "$E2E_IMAGE"
+      - run:
+          name: Build and push E2E << parameters.test_runner >> image
```

Copy-pasted name, should be something like:
```suggestion
          name: Package E2E << parameters.test_runner >> image
```